lab_1/main.py

```diff
@@ -12,14 +12,13 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    tokens = ''
-    text = text.lower()
-    for i in text:
-        if i not in ".,!?^/:;-%@#$^&*()_+=><[]{}\"\'\\":
-            tokens = tokens + i
-    tokens = tokens.split()
 
-    return tokens
+    if text and isinstance(text, str):
+        tokens = re.sub('[^a-zA-Z \n]', '', text).lower()
+        return tokens.split()
+    else:
```

Comment:
you don't need this "else", if "return tokens.split()" works, the function will finish its work